docs/basic-docs/atomicdex-api-20-dev/enable_bch_with_tokens.md

```diff
@@ -0,0 +1,591 @@
+# enable\_bch\_with\_tokens
+
+The AtomicDEX-API supports Bitcoin Cash SLP tokens. Activation of these tokens is a little different to other coins and protocols. Using this method, you can enable multiple SLP tokens in a single command.
```

Comment:
> Activation of these tokens is a little different to other coins and protocols.

You can remove this because we plan to use a similar RPC structure for other protocols. Also, I think it's worth explicitly mentioning that this method activates the platform coin in the first place.